src/vs/workbench/services/configuration/browser/configurationService.ts

```diff
@@ -667,9 +678,9 @@ export class WorkspaceService extends Disposable implements IConfigurationServic
 			.then(() => {
 				switch (editableConfigurationTarget) {
 					case EditableConfigurationTarget.USER_LOCAL:
-						return this.reloadLocalUserConfiguration().then(local => this.onLocalUserConfigurationChanged(local));
+						return this.reloadLocalUserConfiguration().then(() => undefined);
```

Comment:
@sandy081 unclear to me about the consequences, we no longer send configuration change event when writing to config? Wouldn't this break any calls to `writeConfigurationValue`?

Reply:
change event is not removed, it is now trigged in `reloadLocalUserConfiguration`. See here

https://github.com/microsoft/vscode/pull/106726/files#diff-79935821359dac9fd5c4e55b3d11028eR446

https://github.com/microsoft/vscode/blob/6f56451b5985431abc71e5f49ef7429f284e1c47/src/vs/workbench/services/configuration/browser/configurationService.ts#L443-L449